PolyEngine/Engine/Src/HeavyTaskImplementation.hpp

```diff
@@ -0,0 +1,97 @@
+#pragma once
+
+#include "HeavyTaskBase.hpp"
+#include "World.hpp"
+
+namespace HeavyTaskHelper
```

Comment:
Some comment regarding the purpose of the helper would be nice. More than one sentence would be even better ;)

Also, can we move this to namespace Poly?